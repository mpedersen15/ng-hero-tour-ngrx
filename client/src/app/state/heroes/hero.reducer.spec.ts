import { LoadAll } from "./hero.actions";
import { reducer } from "./hero.reducer";

describe('Hero load all reducer', () => {s
    let initialState;
    beforeAll(() => {
        initialState = {
            loading: false
        }
    });

    test('should set loading to true on state', () => {
        const action = new LoadAll(),
        result = reducer(initialState, action);

        expect(result.loading).toMatchSnapshot();
    });
});

describe('Hero load all success reducer', () => {
    let initialState;
    beforeAll(() => {
        initialState = {
            loading: false
        }
    });

    test('should set loading to true on state', () => {
        const action = new LoadAll(),
        result = reducer(initialState, action);

        expect(result.loading).toMatchSnapshot();
    });
});