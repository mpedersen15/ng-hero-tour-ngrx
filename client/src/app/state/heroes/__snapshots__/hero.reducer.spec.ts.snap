// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero load all reducer should set loading to true on state 1`] = `true`;
